home-api-tests/src/test/resources/test_script.sql

```diff
@@ -1,6 +1,6 @@
 INSERT INTO users VALUES(10000000,'2021-06-09 19:53:50.874076',false,'ownertest@example.com',true,false,'test','test',false,'123','2021-06-09 19:53:50.874076');
 INSERT INTO addresses VALUES (10000000, 'test','test','test','test','test','test','test');
-INSERT INTO cooperations VALUES (10000000,'test','11111111','UA111111111111111111111111111','2021-06-09','2021-06-09',true,10000000);
+INSERT INTO cooperations VALUES (10000000,'test','11111111','UA111111111111111111111111111','2021-06-09','2021-06-09',true,10000000, 'test@example.com');
```

Comment:
Why do we need email in cooperation?

Reply:
To create a unique constraint of admin_email and usreo inside of cooperation